charts/nfs-subdir-external-provisioner/templates/role.yaml

```diff
@@ -9,6 +9,9 @@ rules:
   - apiGroups: [""]
     resources: ["endpoints"]
     verbs: ["get", "list", "watch", "create", "update", "patch"]
+  - apiGroups: ["coordination.k8s.io"]
```

Comment:
Can you elaborate on why this is needed?

Reply:
as far as i remember it is part of new leader election in sig-storage-lib-external-provisioner